src/System.Collections/tests/BitArray/BitArray_GetSetTests.cs

```diff
@@ -379,6 +380,24 @@ public static void CopyTo_Int_Hidden(string label, BitArray bits)
         }
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Desktop Framework hasn't received the fix yet.")]
+        [MemberData(nameof(CopyTo_Hidden_Data))]
+        public static void CopyTo_Int_Hidden_Desktop(string label, BitArray bits)
+        {
+            int allBitsSet = unchecked((int)0xffffffff); // 32 bits set to 1 = -1
+            int fullInts = bits.Length / BitsPerInt32;
+            int remainder = bits.Length % BitsPerInt32;
+            int arrayLength = fullInts + (remainder > 0 ? 1 : 0);
+
+            int[] data = new int[arrayLength];
+            ((ICollection)bits).CopyTo(data, 0);
+
+            Assert.All(data, d => Assert.Equal(allBitsSet, d));
+            
+        }
+
+        [Theory]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "Desktop Framework hasn't received the fix yet.")]
```

Comment:
I'm probably spacing but aren't these backwards -- we want CopyTo_Byte_Hidden to skip on .NetFramwework (which is desktop) and vice versa.